.travis.yml

```diff
@@ -5,6 +5,7 @@ rvm:
 - 2.1.5
 - 2.2.0
 - 2.3.0
+- jruby-9.1.5.0
```

Comment:
I agree with your thoughts but today we don't say we support only JRuby x, y or z. Based on the investigation on JRuby you mentioned that "_incompatibility between standard ruby and jruby arises only if the ruby gems use native extensions_". 

With that regards, my thoughts were 

1. as today we don't have any native dependencies we are compatible with JRuby and any future changes in JRuby should not break us either. 
2. checks with head would help us detecting before hand rather than at the time of release of our sdk only. 

With these thoughts in mind, I am more inclined for us to be not bounded by the JRuby versions & keep testing the `jruby-head`

@sarangan12 & @veronicagg what do you guys think?